### ISSUE TITLE
macOS issue: Detaching native window such as QVulkanWindow causes a huge memory leak.

### ISSUE THREAD

**Issue by Servall4** [githubuser0xFFFF/Qt-Advanced-Docking-System#603]
When adding a native window as a tab, a large memory leak occurs (from ~200MB to 1.5GB RAM). I managed to solve this problem by setting this attribute: `QWidget::setAttribute(Qt::WA_DontCreateNativeAncestors)` to window container. 

Now native windows are not created for all widgets in the application. But when I undock this widget into floating window, the RAM again becomes about 1.5GB. I suspect this may also be due to the creation of native ancestors. 

Do you have any idea why this is happening? I think the problem is somewhere on the surface. I'm using Qt 6.5.2.

**Comment by githubuser0xFFFF** [comment]
No, sorry - no idea. I have no Mac and also the number of Mac ADS user is very small. So you are at your own fixing this issue.

**Comment by imikejackson** [comment]
Our cross platform app uses QtADS but we are using OpenGL with VTK. We are using Qt 6.5.x for the gui. We have not seen anything

**Comment by githubuser0xFFFF** [comment]
I do not think, that this is an ADS issue and I cannot reproduce it because I have no Mac.
